pennylane_lightning/core/_version.py

```diff
@@ -16,4 +16,4 @@
 Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.41.0"
+__version__ = "0.41.1rc0"
```

Comment:
It should be safe as we don't push any wheels to PyPI or TestPyPI on PRs.

I'll update this after testing the wheels!

```suggestion
__version__ = "0.41.1"
```